mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraUserMailboxRightsDAO.java

```diff
@@ -98,12 +99,15 @@ private PreparedStatement prepareSelectAllForUser(Session session) {
             .where(eq(USER_NAME, bindMarker(USER_NAME))));
     }
 
-    public CompletableFuture<Void> update(CassandraId cassandraId, PositiveUserACLChanged positiveUserAclChanged) {
+    public ACLDiff update(CassandraId cassandraId, PositiveUserACLChanged positiveUserAclChanged) {
```

Comment:
Shouldn't it be a CompletableFuture<ACLDiff> ?

Reply:
of course